.devcontainer/docker-compose.yml

```diff
@@ -35,6 +35,8 @@ services:
       KONG_PG_HOST: db
       OPENSSL_DIR: /usr/local/kong
       CRYPTO_DIR: /usr/local/kong
+      # env vars to be able to run tests
+      KONG_TEST_PG_DATABASE: kong
```

Comment:
Same here.

Reply:
~Same comment, this is part of VSCode's `.devcontainer` setup.
I think running tests is part of development.